src/main/java/frc/robot/motors/SparkMaxWrapper.java

```diff
@@ -0,0 +1,111 @@
+package frc.robot.motors;
+
+import com.revrobotics.sim.SparkMaxSim;
+import com.revrobotics.spark.SparkMax;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.RobotController;
+import frc.robot.Robot;
+
+/**
+ * Spark max wrapper class, using SparkMaxSim.
+ */
+public class SparkMaxWrapper extends SparkMax implements LoggedMotor {
+
+	// Components
+	private final DCMotor gearbox;
+	private final SparkMaxSim motorSim;
+
+	public static final double LOOP_PERIOD_MS = 0.020;
+
+	private double targetVelocity = 0;
+
+	/**
+	 * Constructor for CAN ID and motor type.
+	 * @param deviceId the CAN ID of the motor
+	 * @param type the type of motor
+	 */
+	public SparkMaxWrapper(int deviceId, MotorType type) {
```

Comment:
If I'm not mistaken MotorType is the Spark MAX [MotorType](https://codedocs.revrobotics.com/java/com/revrobotics/spark/sparklowlevel.motortype) right? Since we're hardcoding a neo motor below, shouldn't this be always brushless?